metagraph/src/graph/alignment/aligner_labeled.cpp

```diff
@@ -11,6 +11,11 @@ namespace align {
 
 using mtg::common::logger;
 
+class early_term : public std::exception {
+  public:
+    const char* what() const noexcept override { return "Finished iteration"; }
+};
+
```

Comment:
I think it would be much better to define it just once somewhere, rather than doing it over and over again in different files.

Reply:
I moved it to `sequence_graph.hpp`. @hmusta Let me know if it's better to put in some other place.